requirements.txt

```diff
@@ -1,12 +1,13 @@
-Flask==1.0.2
+black==19.3b0
```

Comment:
it looks like you sorted this alphabetically.

This doesn't appear to have caused an issue, however it installs and resolved dependencies in order so sometimes you have to place them in a non-aphabetical order to have the correct dependency installed...

Reply:
I did a clean and build and it doesn't seem to have caused any issues.